symengine/series_flint.cpp

```diff
@@ -138,8 +138,8 @@ unsigned URatPSeriesFlint::ldegree(const fqp_t &s)
 {
     long i = 0;
     while (i <= s.degree())
-        if (not s.get_coeff(i++).is_zero())
-            return i - 1;
+        if (not((s.get_coeff(i++)).is_zero()))
```

Comment:
I didn't notice that it was. I'll update this. BTW, can you also enable the auto-cancellation of Travis Builds?